go/vt/vtgate/vindexes/vschema_test.go

```diff
@@ -234,6 +234,14 @@ func init() {
 	Register("mcfu", newMCFU)
 }
 
+func buildVSchema(source *vschemapb.SrvVSchema) (vschema *VSchema) {
+	vs := BuildVSchema(source)
+	if vs != nil {
+		vs.ResetCreated()
+	}
+	return vs
+}
+
```

Comment:
`buildVschema` creates a new VSchema, does it require a `ResetCreated`?

Reply:
As mentioned above we need to zero out `created`, otherwise proto compares fail.